src/Service.Tests/SqlTests/GraphQLQueryTests/GraphQLQueryTestBase.cs

```diff
@@ -1450,6 +1458,92 @@ public virtual async Task TestInvalidFilterParamQuery()
             SqlTestHelper.TestForErrorInGraphQLResponse(result.ToString());
         }
 
+        /// <summary>
+        /// Test to check that sourceFields and targetFields for relationship provided in the config
+        /// overrides relationship fields defined in DB.
+        /// In this Test the result changes when we override the source and target fields in the config.
+        /// </summary>
+        [TestMethod]
+        public async Task TestConfigTakesPrecedenceForRelationshipFieldsOverDB(
+            string[] sourceFields,
+            string[] targetFields,
+            int club_id,
+            string club_name,
+            DatabaseType dbType,
+            string testEnvironment)
+        {
+            RuntimeConfig configuration = ConfigurationTests.InitBasicRuntimeConfigWithNoEntity(dbType, testEnvironment);
```

Comment:
`InitBasicRuntimeConfigWithNoEntity` is a test helper method so better placement would be in `SqlTestHelper`